### DISCUSSION TITLE
Minify livewire.js

### DISCUSSION THREAD

**Discussion by dips83** [livewire/livewire#7620] [1. Help] [answered]
Still learning to use livewire. While trying to improve performance, I noticed that the livewire.js file that created is not minified even in production. Does anyone have a solution for how to do this?

**Comment by vitalijalbu** [comment]
nice idea

**Comment by N1ebieski** [comment]
https://livewire.laravel.com/docs/installation#manually-bundling-livewire-and-alpine

**Comment by kaburagisec** [comment]
The minified version is available in /your-laravel-project/vendor/livewire/livewire/dist, with the file name livewire.min.js

**Reply by sammyaxe** [reply]
but why @livewireScripts directive never serves that?

**Reply by kaburagisec** [reply]
I can't answer that, because I didn't make livewire lol

**Reply by alihaiderx** [reply]
If you use APP_DEBUG=true in your Laravel project, it uses un-minifed version of livewire. If APP_DEBUG is set to false it uses minified version.

**Reply by sammyaxe** [reply]
@MrADevelopers that is the case only in newer versions, as you can see @joshhanley created PR to fix it below.

**Comment by joshhanley** [comment, marked as answer]
I've submitted a PR #7928 with a fix for this.
